src/components/campaigns/grid/CampaignCoordinatorSelect.tsx

```diff
@@ -0,0 +1,33 @@
+import { FormControl, FormHelperText, InputLabel, MenuItem, Select } from '@mui/material'
+import { TranslatableField, translateError } from 'common/form/validation'
+import { useCoordinatorsList } from 'common/hooks/coordinators'
+import { useField } from 'formik'
+import { useTranslation } from 'react-i18next'
+
+export default function CoordinatorSelect({ name = 'coordinatorId' }) {
```

Comment:
Please rename the file to `CoordinatorSelect` as the component inside

There's no reason to be `CampaignCoordinatorSelect`